src/one_dragon/devtools/python_launcher.py

```
@@ -71,18 +72,29 @@ def configure_environment(ctx: OneDragonEnvContext, cwd):
+    uv_path = ctx.env_config.uv_path
+    app_module_parts = app_path.copy()
+    module_name = app_module_parts[-1]
+    if module_name.endswith('.py'):
+        module_name = module_name[:-3]
+    app_module_parts[-1] = module_name
+    app_module = '.'.join(app_module_parts)
+
+    app_module_path = Path(os.environ.get('PYTHONPATH', '')).joinpath(*app_module_parts)
+    app_file_path = app_module_path.with_suffix('.py')
+    if not app_file_path.is_file():
+        print_message(f"PYTHONPATH 设置错误，无法找到 {app_file_path}", "ERROR")
```

Ruff (0.15.13)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)